build.xml

```diff
@@ -22,18 +23,339 @@
+            </test>
+            -->
+			<!--
+            <test name="org.scada_lts.service.ViewHierarchyServiceMoveFolderTest"
+                haltonfailure="yes">
+            </test>
+
+            <test name="org.scada_lts.service.ViewHierarchyServiceMoveViewTest"
+                haltonfailure="yes">
+            </test>
+            -->
+			<test name="org.scada_lts.cached.DataPointsCacheComposeDataSourcesDataTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.cached.DataSourcePointsCacheMockitoTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueCacheTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueFacadeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.web.dwr.DataPointDetailsDwrTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueTimeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.AnnotatedPointValueTimeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.dao.PointValueDAOTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.view.component.ViewComponentTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataSource.meta.ScriptExecutorTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.scripting.ContextualizedScriptRTTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.ScheduledExecuteInactiveEventServiceTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.vo.mailingList.MailingListInactive15MinuteIntervalTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.event.handlers.EmailToSmsHandlerRtTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.CommunicationChannelTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.event.ScheduledExecuteInactiveEventRtTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.InactiveEventsProviderTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.utils.UpdateValuePointPropertiesTest"
+				  haltonfailure="yes">
+			</test>
+		</junit>
+		<record action="stop" name="${junit.log.name}"/>
+
+	</target>
+
+	<target name="test-frontend-unit" description="Test Scada-LTS Frontend application">
+		<exec dir="./scadalts-ui" executable="npm.cmd" osfamily="windows">
+			<arg value="run-script"/>
+			<arg value="test:unit"/>
+		</exec>
+		<exec dir="./scadalts-ui" executable="npm" osfamily="unix">
+			<arg value="run-script"/>
+			<arg value="test:unit"/>
+		</exec>
+	</target>
+
+	<target name="create-war" description="Build and deploy WAR archive.">
```

Comment:
depends="build-skip-test"

Reply:
`create-war` is a basic task that is a part of the `deploy` task group.  
Basic task should not depends on any task. To perform 'create-war' that depends on `"build-skip-test"` use `"build-skip-test-run"`
See the build.xml file because it is already done in configuration. Description is provied in "readme.md" file and in the PullRequest description.

Reply:
Same as in the previous comments. It is worth finding a compromise between independence from other targets and autonomy in launching them.